src/pages/Swap/index.tsx

```diff
@@ -441,6 +441,11 @@ export function Swap({
         })
       })
       .catch((error) => {
+        if (!didUserReject(error)) {
+          sendAnalyticsEvent(SwapEventName.SWAP_ERROR, {
+            confirmedTrade: tradeToConfirm,
```

Comment:
are you sure that sending a full object will be okay here? if yes then all good

Reply:
there should be a way to test sending into the amplitude dev environment if you want to try that. i worry you won't be able to filter properly this way

Reply:
https://analytics.amplitude.com/uniswap/project/396672/search/amplitude_id%3D525267312650?sessionHandle=YjFTPAt_YjFTPgP_HpMXyAK_Bg2A_19308d93-cd11-4971-961e-fc55a0f758c9&eventId=e2ba7448-8ff4-4fdd-9ba8-9c1f3c739a85

yes it seems fine! checked with blair too